webhook.rst

```diff
@@ -16,8 +19,89 @@ Installation
 
     $ composer require symfony/webhook
 
+
+Consuming Webhooks
+------------------
+
+Consider an example of an API where it's possible to track the stock levels of various products.
+A webhook has been registered to notify when certain events occur, such as stock depletion for a specific product.
+
+During the registration of this webhook, several pieces of information were included in the POST request,
```

Comment:
This is information that is typically available in the 3rd party service documentation?